test/runtests.jl

```diff
@@ -3,3 +3,5 @@ include("test_Dogs.jl")
 include("test_Sheeps.jl")
 include("test_DenseSheeps.jl")
 include("test_Simulation.jl")
+include("test_measures.jl") 
```

Comment:
```suggestion
include("test_Measures.jl") 
```
would be good to be consistent here